Makefile

```diff
@@ -57,23 +59,8 @@ TEST_ENVS := chop_test crush_test degree_test depth_test flip_test \
 slow-odgi-tests:
 	turnt -j $(TEST_ENVS:%=--env %) test/*.gfa
 	turnt -j --env validate_test_err test/invalid/*.gfa
-
-# The basic test suite above, plus a few handmade tests for good measure.
-# Those are described below.
-test-slow-odgi-careful: slow-odgi-crush-careful slow-odgi-flip-careful
```

Comment:
I had fenced these off as "careful" because they are small handmade tests that are designed to test specific nuances in the odgi specs. Someone could roll their eyes and call them _overly_ careful. I'm all for including them in the main test flow if you are!

Reply:
Cool; sounds right to me! In a weird way, "testing weird nuances" is kinda the _normal_ role that tests play; our other approach (use a sampling of real-world inputs and hope they cover interesting cases naturally) is less standard. So yeah, I'm all for including our extra-pedantic, targeted tests as part of the normal workflow.